### PR TITLE
Feature/conditionals

### PR DESCRIPTION
**Pull request by Hawawou**
fetching data from json file

### CODE REVIEW

File: frontend/public/data.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    },
+    {
+      "id": 6,
+      "text": "Vision",
+      "type": "Q",
+      "children": []
+    },
+    {
+      "id": 7,
+      "text": "Promote talent and kknow-how of Cameroonian by using our technical knowledge to ring solutions to problems found in our society ",
```
Comment (elhmn) [inline frontend/public/data.json]: a typo here ?
```suggestion
      "text": "Promote talent and know-how of Cameroonian by using our technical knowledge to ring solutions to problems found in our society ",
```

File: frontend/src/components/PrevNext.js [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
+    })
+    .then(function(response){
+      console.log(response)
+      return response.json();
+     })
+     .then(function(myJson){
+       console.log(myJson);
+       setData(myJson)
+     });
   }
```
Comment (elhmn) [inline frontend/src/components/PrevNext.js]: Why are we doing an http request to get a local json file here, can't we read the file instead ?

File: frontend/src/components/Body.js [hunk 3]
```diff
@@ -1,43 +1,75 @@
...
+            this.setState({get_child: response.data.children})
+        })
+        .catch(error => {
+            console.log(error)
+        })
       }
+
+       handleClick() {
+       //console.log("Hello")
+       // if ({child.id} = 1){
```
Comment (elhmn) [inline frontend/src/components/Body.js]: Maybe we should avoid leaving commented code in our PRs

File: frontend/src/components/Body.js [hunk 4]
```diff
@@ -1,43 +1,75 @@
...
+     // }
+       
+         
+       
+       }
       
     render() {
         const {get_child} = this.state
         console.log("get_child:", get_child);
+       // const {show_child} = this.state.show_child;
```
Comment (elhmn) [inline frontend/src/components/Body.js]: this should not be in a PR

File: frontend/src/App.js [hunk 5]
```diff
@@ -12,7 +13,9 @@ function App() {
     <div className="App">
        <Exit />
        <Header text = {'Welcome to hawaBot. The official FAQ bot for Oss Cameroon. We provide an answer to all your questions about OssCameroon!'}/>
-       <Body />
+       
+       <PrevNext />
+
```
Comment (elhmn) [inline frontend/src/App.js:19]: Maybe you can remove this empty lines

File: frontend/src/components/Body.js [hunk 6]
```diff
@@ -1,43 +1,75 @@
...
+            <center><button type='Submit' className="btn-class" onClick={() => {this.handleClick()}} >
+            Founders of Oss Cameroon
+            </button> </center>
+         </div>
           <div>
+          
+         { this.state.show_child ?  
+              
+                get_child && get_child.length?
+                get_child.map(child => <div key={child.id}  onClick={() =>{ this.getChildrenByParentID(child.id)}}>{parse(child.text)} </div> ): null
```
Comment (elhmn) [inline frontend/src/components/Body.js]: Can this code be extracted in a functional component ? Using nested ternaries does not make code very easy to understand

**Review by elhmn — CHANGES_REQUESTED** [review]
Thank you for taking care of that, I left you few comments and questions about the PR

File: frontend/src/components/PrevNext.js (hunk 2 above)
Comment (Hawawou) [reply frontend/src/components/PrevNext.js]: Actually I can

File: frontend/src/components/Body.js (hunk 4 above)
Comment (Hawawou) [reply frontend/src/components/Body.js]: This file is no longer useful. I'll delete it

**Review by elhmn — APPROVED** [review]

**Pull request merged**